root/etc/services.d/kodi/run

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/with-contenv bash
 
 	s6-setuidgid abc \
-	/usr/bin/kodi --headless --debug \
+	/usr/bin/kodi --headless \
 	> /config/.kodi/kodi.output.log 2>&1
```

Comment:
does Kodi not automatically create a log file?

Reply:
I guess you are referring to stdout redirection !?
This is because In the patch I replaced the original Kodi user notifications sent to splash screen with stdout print !
It could either be removed from the code either be dropped !?

It was for me interesting to keep these user oriented messages not mixed with technical logs.

Reply:
Ok, I see... Sorry I haven't inspected your patch so deeply...

But your idea sounds good 👍
Maybe the log file can be renamed to `kodi.screen_output.log` or something similar? Just to see at a glance what this file contains ;)

Reply:
Or if kodi is started with the `--headless` flag, this log file gets automatically created like the normal `kodi.log`? Just a quick idea...

Reply:
Done!
Other and may be more elegant option would be to set a docker env variable to activate/deactivate debugs !?
To be considered later...